src/components/Person/Person.jsx

```diff
@@ -1 +1,36 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const getPartner = () => {
+    if (isMarried) {
+      return sex === 'm'
+        ? 'wife'
+        : 'husband';
+    }
+
+    return 'I am not married';
+  };
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age && <p className="Person__age">{`I am ${age}`}</p>}
+
+      <p className="Person__partner">
+        {partnerName
+          ? `${partnerName} is my ${getPartner()}`
+          : getPartner()
```

Comment:
I think will be better to refactor this solution and remove `getPartner` function.

Instead of calling `getPartner` the second time, we can use 'I am not married' string.